### ISSUE TITLE
Email notification of other tenants have wrong links

### ISSUE THREAD

**Issue by Knorrke** [serlo/athene2#912]
e.g. email contains link `https://www.serlo.org/entity/view/41648` which redirects to `de.` instead of `en.`

Interestingly `https://www.serlo.org/41648` works, as `https://de.serlo.org/41648` tries to resolve the correct instance

**Comment by inyono** [comment]
To be honest, I'm not even sure where / how `https://www.serlo.org/FOOBAR` gets resolved because locally it doesn't work at all. (And like I wrote you on Slack, the Cloudflare Worker currently only redirects root).
If there's no special treatment for `www.serlo.org` / `serlo.org`, I propose that the alias module redirects to another tenant if the specific alias doesn't exist in this tenant (do we even allow en.serlo.org/FOO and de.serlo.org/FOO simultaneously at the moment?)

**Comment by Knorrke** [comment]
Yes, the aliases can be the same on different tenants e.g. /serlo, but ids shouldn't..
